src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,74 @@
+package calculator
+
+class Calculator {
+    fun plusOperation(
+        number1: Float,
+        number2: Float,
+    ): Float {
+        return number1 + number2
+    }
+
+    fun minusOperation(
+        number1: Float,
+        number2: Float,
+    ): Float {
+        return number1 - number2
+    }
+
+    fun multipleOperation(
+        number1: Float,
+        number2: Float,
+    ): Float {
+        return number1 * number2
+    }
+
+    fun divideOperation(
+        number1: Float,
+        number2: Float,
+    ): Float {
+        if (number2.toInt() == 0) throw Exception("Not divided by 0")
+        return number1 / number2
+    }
+
+    fun isOperator(input: String): Boolean {
+        val operators = listOf("+", "-", "*", "/")
+        return operators.any { it == input }
+    }
+
+    fun splitEquation(equation: String): List<String> {
+        val regex = OPERATION_REGEX.toRegex()
+        return regex.findAll(equation).map { it.value }.toList()
+    }
+
+    fun getResult(equation: String?): Float {
```

Comment:
```suggestion
    fun getResult(equation: String): Float {
```

코틀린은 Nullable과 Null이 확실한 언어죠. 그렇다면, 처음부터 Nullable하게 들어오지 못하게 만든다면, 함수 내에서 NullCheck가 필요 없지 않을까요?